retreaver-typescript/src/base/RequestNumber.ts

```
@@ -0,0 +1,260 @@
+  private config: RequestNumberOptions;
+  private resource_url: string = '/api/v1/numbers?';
+
+  constructor(options: RequestNumberOptions) {
+    this.config = Helpers.assertRequiredKeys(options, 'campaign_key');
+  }
+
+  /**
+   * Perform the number request
+   * @param callback - Callback to fire after request
+   */
+  perform(callback: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)